packages/manager/apps/vrack-services/src/components/EditableText.tsx

```diff
@@ -34,8 +38,10 @@ export const EditableText: React.FC<EditableTextProps> = ({
   disabled,
   type = ODS_INPUT_TYPE.text,
   onEditSubmitted,
-  dataTracking,
+  dataTrackingPath,
+  editDataTracking,
   confirmDataTracking,
+  trackClick,
```

Comment:
Shouldn't we import the hook directly here instead of passing it with props ?

Reply:
It's not possible untill we switch to React Datagrid because the cells of the ODS datagrid don't have the context